package.json

```diff
@@ -3,16 +3,32 @@
   "version": "0.1.0",
   "private": true,
   "dependencies": {
+    "@auth0/auth0-react": "^1.8.0",
```

Comment:
Let's have a "devDependencies" section in our package.json for test related modules :)